//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// abcdef_T_V.h
//
// Code generation for function 'abcdef_T_V'
//

#ifndef ABCDEF_T_V_H
#define ABCDEF_T_V_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void abcdef_T_V(double V_init, double A_init, double V_wayp, double V_max,
                double A_max, double A_min, double J_max, double J_min,
                double T, double t[7]);

#endif
// End of code generation (abcdef_T_V.h)
